template-for-repository/proofs/Makefile.common

```diff
@@ -83,6 +83,9 @@ default: report
 # Absolute path to the directory containing this Makefile.common
 # See https://ftp.gnu.org/old-gnu/Manuals/make-3.80/html_node/make_17.html
 PROOF_ROOT = $(dir $(abspath $(filter %/Makefile.common,$(MAKEFILE_LIST))))
+HELPER_DIR = $(shell dirname $(PROOF_ROOT))
+STUBS_DIR = $(HELPER_DIR)/stubs
+MODELS_DIR = $(HELPER_DIR)/models
```

Comment:
Again, I'd use PROOF_STUB, PROOF_SOURCE, PROOF_INCLUDE.

HELPER_DIR is a name we commonly use, but it is pretty meaningless, and I intentionally did not use it.  I'd rather something like CBMC_ROOT for cbmc, and have PROOF_STUB, PROOF_SOURCE, PROOF_INCLUDE be the stubs, sources, include subdirs.

Reply:
I changed HELPER_DIR to CBMC_ROOT.
I changed STUBS_DIR to PROOF_STUB.
I changed MODELS_DIR to PROOF_SOURCE and renamed "models" back to "sources".

I did not add PROOF_INCLUDE because we don't know what the include order will be. The recommendation in the `Makefile-project-defines` file is now to add `-I$(CBMC_ROOT)/include` to the `INCLUDES` variable. I could set `PROOF_INCLUDE = $(CBMC_ROOT)/include` and recommend to add `PROOF_INCLUDE` to `INCLUDE`, but it seems unnecessary to create a variable that will only ever be used once (we will never append to `PROOF_INCLUDE` and never use it in any other context).